src/QsCompiler/TestProjects/test10/Driver.cs

```diff
@@ -1,13 +0,0 @@
-﻿using Microsoft.Quantum.Simulation.Core;
-using Microsoft.Quantum.Simulation.Simulators;
-
-namespace test3
-{
-    class Driver
-    {
-        static void Main(string[] args)
-        {
-
-        }
-    }
-}
```

Comment:
Is this file intentionally deleted as part of the version changes?

Reply:
I believe this is intentional as part of clean up. I don't think this Driver.cs is needed anymore. @kuzminrobin if this is a mistake.

Reply:
This file is needed if you try to open the .csproj and compile the project. 
But the test does not copy this file to the test working dir (see the [copying fragment](https://github.com/microsoft/qsharp-compiler/blob/e09f8d7db23a07a3f896e9903d0b06c0452d2b94/src/QsCompiler/Tests.LanguageServer/Tests.LanguageServer.csproj#L33-L40)), the test never tries to compile the test project. This file is never used.

Reply:
Look at the 'test10' project, the .qs file contains barely anything it anyway. I'm not sure the value of this project/test. Fine I guess.

Reply:
Those test projects, I believe, are tested against _loadability_, not _compilability_ or _runnability_. Their main files are .csproj ones (the .qs ones are just for completeness/consistency).